src/classmentors/components/session/mcq/mcq.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Created by AMOS on 18/6/16.
+ */
+(function(){
```

Comment:
You do not need for an Immediately-Invoked Function Expression; es6 module context is not the global context. There no risk to exporting/overwrite `window`.